driver/main.c

```diff
@@ -1626,92 +1603,21 @@ static inline void record_drop_x(struct ppm_consumer_t *consumer,
 	}
 }
 
-// Return 1 if the event should be dropped, else 0
```

Comment:
Dropped dropping logic based on event type. Really useless considering that one can now completely disable syscalls through syscalls of interest mechanism.

Reply:
Same as above, this is a finer-grained mechanism; see the comment below (lines 1625-1630) for a rationale.